code/Controllers/SilverStripeNavigatorItem_CMSLink.php

```diff
@@ -16,13 +17,13 @@ public function getHTML()
         return sprintf(
             '<a href="%s">%s</a>',
             $this->record->CMSEditLink(),
-            _t('SilverStripe\\CMS\\Controllers\\ContentController.CMS', 'CMS')
+            _t(ContentController::class . '.CMS', 'CMS')
         );
     }
 
     public function getTitle()
     {
-        return _t('SilverStripe\\CMS\\Controllers\\ContentController.CMS', 'CMS', 'Used in navigation. Should be a short label');
+        return _t(ContentController::class . 'CMS', 'CMS', 'Used in navigation. Should be a short label');
```

Comment:
```suggestion
        return _t(ContentController::class . '.CMS', 'CMS', 'Used in navigation. Should be a short label');
```

Reply:
Good catch, I'll fix that tomorrow

Reply:
Reverted instead per https://github.com/silverstripe/silverstripe-cms/pull/2761#discussion_r970544389